packages/buendia-site-cloud-demo/data/usr/share/buendia/site/40-site

```diff
@@ -16,4 +16,7 @@ MONITORING_LIMITS='
 100000 /usr/share/buendia/packages
 '
 
+# Enable automatic upgrades and forced upgrades, so that we always have the
+# latest code running.
```

Comment:
On demo, we don't want demo data to be blown away without manual intervention; forced upgrade should be possible (and easy) but not automatic.